src/krux/pages/__init__.py

```diff
@@ -195,15 +289,14 @@ def display_qr_codes(
             subtitle = (
                 t("Part\n%d / %d") % (i + 1, num_parts) if title is None else title
             )
-            offset_y = 175 if title is None else 138
-            self.ctx.display.draw_hcentered_text(
-                subtitle, offset_y=offset_y, color=lcd.WHITE, padding=title_padding
-            )
+            offset_y = self.ctx.display.qr_offset()
+            if title is not None:
+                offset_y += self.ctx.display.font_height
+            self.ctx.display.draw_hcentered_text(subtitle, offset_y, color=lcd.WHITE)
             i = (i + 1) % num_parts
-            btn = self.ctx.input.wait_for_button(block=num_parts == 1)
-            done = btn == BUTTON_ENTER
-            if not done:
-                time.sleep_ms(QR_ANIMATION_INTERVAL_MS)
+            if self.ctx.input.wait_for_proceed():
```

Comment:
`wait_for_proceed` needs to be updated to take an optional `block=True` param like `wait_for_button`,  and the same expression should be passed to it as was here before:
`if self.ctx.input.wait_for_proceed(block=num_parts == 1)`

Otherwise, QR codes will stop animating automatically